torch/csrc/jit/python/script_init.cpp

```diff
@@ -1877,6 +1917,10 @@ void initJitScriptBindings(PyObject* module) {
   m.def("_test_only_populate_upgraders", &test_only_populate_upgraders);
   m.def("_test_only_remove_upgraders", &test_only_remove_upgraders);
 
+  m.def("_create_argument", &createArgument);
+  m.def("_create_function_schema", &createFunctionSchema);
```

Comment:
Can you just expose the FunctionSchema constructor to Python instead of adding a `_create_function_schema` function?

Reply:
Same for create_argument and create_alias_info. and create_tuple_type.

Reply:
Sure